modules/v2x/v2x_proxy/proto_adapter/BUILD

```diff
@@ -0,0 +1,34 @@
+load("//tools:cpplint.bzl", "cpplint")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "v2x_proxy_proto_adapter",
+    srcs = ["proto_adapter.cc"],
+    hdrs = ["proto_adapter.h"],
+    deps = [
+        "//modules/v2x/common:v2x_proxy_gflags",
+        "//modules/v2x/proto:v2x_carstatus_cc_proto",
+        "//modules/v2x/proto:v2x_rsi_cc_proto",
+        "//modules/v2x/proto:v2x_traffic_light_cc_proto",
+        "//modules/v2x/proto:v2x_obu_traffic_light_cc_proto",
+        "//modules/v2x/proto:v2x_obu_rsi_cc_proto",
+        "//modules/v2x/proto:v2x_junction_cc_proto",
+        "//modules/map/hdmap",
+        "//cyber",
+        "@eigen",
+    ],
+)
+
+cc_test(
+    name = "proto_adapter_test",
```

Comment:
This one is good!

Don't forget to format the BUILD file with

```
scripts/buildifier.sh modules/v2x/v2x_proxy/proto_adapter/BUILD
```

Or  `apollo.sh format modules/v2x/v2x_proxy`